modin/pandas/io.py

```diff
@@ -368,6 +615,29 @@ def read_excel(
 ) -> "DataFrame | dict[IntStrT, DataFrame]":  # noqa: PR01, RT01, D200
     """
     Read an Excel file into a DataFrame.
+
+    Parameters Support Status:
+
+    +-------------------------+-----------------+----------------+----------------+----------------------------------+
+    | Parameters              | PandasOnRay     | PandasOnDask   | OmniSci        | Notes                            |
+    +=========================+=================+================+================+==================================+
+    | All parameters          | Partial         | Partial        | Harmful        | OmniSci execution is not         |
```

Comment:
Here we have "All parameters" line somewhat redundant since it duplicates info from other lines?

Reply:
Yes, in some cases like this `All parameters`/Summary row can be redudant. Alternatively we could parse parameters of table and conclude summary flag for each of executions, but in some corner cases it can be impossible to make this conclusion without the knowledge of the function signature (only docstring are parsed in #4286 for now).
For exampe, some function has 3 parameters - a,b and c, and in the docstring table only a and b are present with `Harmful` flag (parameter c is fully supporter and not present in the table), but according to the table all parameters are `Harmful` and we can conclude (without extra function signature knowledge) that `Harmful` value should used as summary for this function, while it should be `Partial`. To fix this issue we should additionally parse function signature and make an algorithm to determine `All parameters` row according to the present parameters data in the docstring table. It is possible, but seems to me more error prone and less maintainable rather than predefined by developer `All parameters` values. @Garra1980 what do you think?

Reply:
Am I correct that for this particular function for Ray/Dask engines we have 3 params supported partially and other params are fully supported while for Omnisci the whole function is unsupported?

Reply:
> Am I correct that for this particular function for Ray/Dask engines we have 3 params supported partially and other params are fully supported while for Omnisci the whole function is unsupported?

Yes, this is fully correct.

Reply:
Ok, let's leave as is now - hopefully this will be changed after we add coverage for Omnisci engine

Reply:
> parameter c is fully supporter and not present in the table

Why may c be not present in the table? Shouldn't we put `Supported` flag for such a parameter.

Reply:
We shouldn't put `Supported` flag as parameter description, becasuse by convention proposed [here](https://github.com/modin-project/modin/pull/4286#issuecomment-1058098044) we should put in the table only information about unsuppored parameters to avoid table bloating.

Reply:
Yes, we have note in the header of source files containing supported parameters tables ([link](https://github.com/modin-project/modin/pull/4402/files#diff-506c226c0b2753ce3bbf33ea6c4de22796376d982b483a4d9bbbb3e3e6561a9dR40-R44)).

Reply:
Ok, thanks!

Reply:
I think we should put that note in supported APIs pages too and provide more elaborated explanation what Supported APIs table is and what Parameters Support Status table is.

Reply:
Yes, sure, we can add explanation in the Supported APIs section too.

Reply:
I am thinking about that it would probably be nicer and better to support the tables in supported APIs pages, rather than duplicate the table in each file. We could put one or two sentence on how methods' docstrings are organized and add links to `Parameters Support Status` table and `Flags meaning` table in each file. What do you think?

Reply:
I think such duplication is not critiacal and we can leave it for now. And once #4286 is done, we can can refer to the Supported APIs section as you suggested.

Reply:
ok